website/src/pages/Status/Status.jsx

```diff
@@ -52,7 +59,29 @@ const Status = () => {
           </span>
         </div>
 
-        <figure className="statusStats"></figure>
+        <div className="statusStats flex--column">
+          <CountUp
+            start={0}
+            end={dataStatusStats.Benchmark_total}
+            style={{ fontSize: "5rem", color: "#E77002" }}
+            duration={3}
+          />
+          <span className="countUp__title">Benchmark Total</span>
```

Comment:
It would be nice to have the top of that div at the same level as the top of the `h2`